### PR TITLE
Refactor CLI template

### PR DESCRIPTION
**Pull request by dshevtsov**
## Purpose of this pull request

This pull request changes the CLI reference topic and template to make the template reusable in topics for other CLI tools.

## Affected DevDocs pages

- the content was not changed

## Additional information

Internal build for preview: 652

### CODE REVIEW

File: guides/v2.3/reference/cli/magento.md [hunk 1]
```diff
@@ -1,11 +1,24 @@
 ---
 layout: full-width
+title: 'Command-line tools: bin/magento'
```
Comment (jeff-matthews) [inline guides/v2.3/reference/cli/magento.md]: Is there a reason why this isn't symlinked to 2.2? It looks identical to the 2.2 file.

File: _includes/reference/cli-template.md [hunk 2]
```diff
@@ -1,17 +1,15 @@
...
-The initial list is auto generated using the `bin/magento list` command at the {{ site.data.var.ce }} edition. Use the ["Add CLI commands"]({{ page.baseurl }}/extension-dev-guide/cli-cmds/cli-add.html) guide to add a custom Magento CLI command.
+This reference contains {{ commands | size }} commands available through the `{{ tool }}` command-line tool.
+The initial list is auto generated using the `{{ tool }} list` command at the {{ edition }} edition.
+
+{{ intro }}
 
-{: .bs-callout-info }
-You can call Magento CLI commands using shortcuts instead of the full command name. For example, you can call `bin/magento setup:upgrade` using `bin/magento s:up`, `bin/magento s:upg`, etc. See [shortcut syntax](https://symfony.com/doc/current/components/console/usage.html#shortcut-syntax) to understand how to use shortcuts with any Magento CLI command.
+{% if app.version %}
+version {{ app.version }}
```
Comment (hguthrie) [inline _includes/reference/cli-template.md]: I think this line is important, but I would like to see it more pronounced or more obvious. Maybe we could create a summary list of sorts. For example:

**Tool**: 
**Version**:

File: guides/v2.3/reference/cli/magento.md (hunk 1 above)
Comment (dshevtsov) [reply guides/v2.3/reference/cli/magento.md]: This uswe different data files, so the following line is different:
```
{% assign file = site.data.codebase.v2_2.open-source.bin-magento %}
```
```
{% assign file = site.data.codebase.v2_3.open-source.bin-magento %}
```

File: _includes/reference/cli-template.md (hunk 2 above)
Comment (dshevtsov) [reply _includes/reference/cli-template.md]: Good point, I didn't finish with 'version'.

**Comment by dshevtsov** [comment]
running tests

**Pull request merged**